app/models/personal_details.rb

```diff
@@ -7,17 +7,7 @@ class PersonalDetails < ApplicationRecord
     title: 4,
```

Comment:
There's a `DATE_OF_BIRTH_LIMIT` at the top of this file

Reply:
Ah, good spot! 👀 Thanks, I'll fix that now.